src/Common/ZooKeeper/IKeeper.h

```diff
@@ -110,6 +110,15 @@ bool isUserError(Error code);
 const char * errorMessage(Error code);
 
 
+enum KeeperApiVersion : uint8_t
+{
+    V0 = 0, // ZooKeeper compatible version
+    V1      // added FilteredList request
+};
+
+inline constexpr auto current_keeper_api_version = KeeperApiVersion::V1;
+inline constexpr auto * keeper_api_version_path = "/keeper-api-version";
```

Comment:
Maybe it worth using some generic path for such requests, and this will became `/__keepeer_api/version` or similar, that way the `keeper` itself can check for the base path instead of each paths, if later some new metrics will be added.

By the way maybe it will more compatible to export it as a new command or report additional statistics in existing API, i.e. `mntr`, right now zookeeper reports it's version there:

```
# nc 127.1 2181 <<<mntr
zk_version      3.4.13-81ad7b000cec0503b9a1d5521fdaf0129443b536, built on 11/29/2020 16:32 GMT
...
```

Reply:
I'll try to do it in that way.